src/main/java/ch/njol/skript/expressions/ExprRepeated.java

```diff
@@ -10,52 +10,64 @@
 import ch.njol.skript.doc.Since;
 import ch.njol.skript.lang.Expression;
 import ch.njol.skript.lang.ExpressionType;
-import ch.njol.skript.lang.SkriptParser;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
 import ch.njol.skript.lang.util.SimpleExpression;
 import ch.njol.util.Kleenean;
+
 @Name("Repeated")
 @Description("A string repeated a number times")
-@Examples({"on chat:", "\tset {_t} to message repeated 3 times", "\tsend {_t}"})
+@Examples({"on chat:",
+	"\tset {_t} to message repeated 3 times",
+	"\tsend {_t}"})
 @Since("INSERT VERSION")
 public class ExprRepeated extends SimpleExpression<String> {
+	
 	static {
 		Skript.registerExpression(ExprRepeated.class, String.class, ExpressionType.COMBINED, "%string% repeat[ed] %number% time[s]", "repeat[ed] %string% %number% time[s]");
 	}
+	
+	@SuppressWarnings("null")
+	private Expression<String> text;
+	
+	@SuppressWarnings("null")
+	private Expression<Long> times;
+	
 	@SuppressWarnings({"unchecked", "null"})
 	@Override
-	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, SkriptParser.ParseResult parseResult) {
+	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
 		text = (Expression<String>) exprs[0];
 		times = (Expression<Long>) exprs[1];
 		return true;
 	}
-	@SuppressWarnings("null")
-	private Expression<String> text;
-	@SuppressWarnings("null")
-	private Expression<Long> times;
+	
 	@Nullable
 	@Override
 	@SuppressWarnings("null")
 	protected String[] get(Event e) {
-		assert times != null && text != null;
 		String string = text.getSingle(e);
+		Integer number = times.getSingle(e).intValue();
```

Comment:
this may still cause an NPE if getSingle returns null - you have to get the intValue on another line after a null check